src/components/Patient/PatientRegistration.tsx

```
@@ -307,558 +288,476 @@
-                    <Label className="mb-2">
-                      {t("year")}
+                    </FormLabel>
+                    <FormControl>
+                      <RadioGroup
+                        {...field}
+                        onValueChange={field.onChange}
+                        value={field.value}
+                        className="flex gap-5 flex-wrap"
+                      >
+                        {GENDER_TYPES.map((g) => (
+                          <FormItem className="flex items-center space-x-2 space-y-0">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)